main/FatesHistoryInterfaceMod.F90

```diff
@@ -3598,12 +3598,12 @@ subroutine define_history_vars(this, initialize_variables)
          ivar=ivar, initialize=initialize_variables, index = ih_fire_intensity_pa )
 
     call this%set_history_var(vname='FIRE_AREA', units='fraction',             &
-         long='spitfire fire area:m2', use_default='active',                    &
+         long='spitfire fire area burn fraction:%', use_default='active',                    &
```

Comment:
@jkshuman , is this in percent? I can't find where the fraction  is multiplied by 100

Reply:
yeah - it's a fraction.

Reply:
@rgknox looks like that was already updated...trying to see why it isn't updated here.

Reply:
@rgknox that fix to the meta data for fire_area as a "burn fraction" is already in there. that section was outdated code. I marked that as resolved.
https://github.com/jkshuman/fates/blob/443cd8a097e3f3e678f8e83a2010d027213740d8/main/FatesHistoryInterfaceMod.F90#L3601